pyomo/contrib/mindtpy/config_options.py

```diff
@@ -33,7 +33,7 @@ def _get_GDPopt_config():
     ))
     CONFIG.declare("strategy", ConfigValue(
         default="OA",
-        domain=In(["OA", "GBD", "ECP", "PSC", "GOA"]),
+        domain=In(["OA", "GBD", "ECP", "PSC", "GOA", "feas_pump"]),
```

Comment:
This is something that we need to thing more. We want to have the feasibility pump as a standalone strategy and also as an initialization procedure. Contrary to the initialization strategies we have in place (rNLP, max_binary) that decide how to deal with the first iteration, we also want to decide if we want to run the feasibility pump before starting OA or ECP. At the very least we can use the initialization, but the paper I wrote shows that the cuts generated in the process are also useful (we can decide this with an option).